### PR TITLE
MINOR: fix log statement

### PR DESCRIPTION
**Pull request by mjsax**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Review by wcarlson5 — APPROVED** [review]
Lgtm

File: streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java [hunk 1]
```diff
@@ -1536,7 +1536,7 @@ private Consumer<StreamThread> streamThreadLeaveConsumerGroup(final long remaini
                     removeMembersFromConsumerGroupResult.memberResult(memberToRemove)
                         .get(remainingTimeMs, TimeUnit.MILLISECONDS);
                 } catch (final Exception e) {
-                    log.error("Could not remove static member {} from consumer group {} due to a: {}",
+                    log.error("Could not remove static member {} from consumer group {} due to:",
                         groupInstanceId.get(),
                         applicationConfigs.getString(StreamsConfig.APPLICATION_ID_CONFIG), e);
```
Comment (cadonna) [inline streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java:1541]: I guess you want to log the exception, right? If yes, a third pair of curly braces is missing, isn't it? Method `error()` has signatures
```
public void error(String format, Object... arguments);
```
and 
```
public void error(String msg, Throwable t);
```
There is no signature that takes a number of arguments and then a `Throwable`. I pretty sure that would be incorrect java.

My suggestion is:
```suggestion
                    log.error("Could not remove static member {} from consumer group {} due to: {}",
                        groupInstanceId.get(),
                        applicationConfigs.getString(StreamsConfig.APPLICATION_ID_CONFIG), 
                        e
                    );
```
or
```suggestion
                    log.error("Could not remove static member {} from consumer group {} due to: {}",
                        groupInstanceId.get(), applicationConfigs.getString(StreamsConfig.APPLICATION_ID_CONFIG), e);
```

Logging the stacktrace would also be great!

**Review by cadonna — COMMENTED** [review]
@mjsax Thanks for  the PR!

Here my feedback!

File: streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java (hunk 1 above)
Comment (mjsax) [reply streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java:1541]: Screen Shot 2023-04-04 at 5 22 07 PM

Hmmm... I was actually just reacting to an IntelliJ warning -- seems IntelliJ has a bug.

Closing this PR.

**Pull request closed without merging**